src/infrastructure/delivery/index.ts

```diff
@@ -0,0 +1,19 @@
+import {PostTemplateMessageBuilder} from "./entities";
+import {Sender as MailchimpSender} from "./entities/mailchimp/Sender";
+
+const APIKEY = "12345";
```

Comment:
It can be nice to have the API key and the template in an environment var.

Reply:
Yes of course. This is only a placeholder :)